static/app/components/events/interfaces/spans/inlineDocs.tsx

```diff
@@ -123,9 +39,183 @@ class InlineDocs extends Component<Props, State> {
+                eat_slice(pizza.slices.pop())
+`}
+      </StyledCodeSnippet>
+      <StyledCodeSnippet language="python" dark>
+        {`import sentry_sdk
+
+@sentry_sdk.trace
+def eat_slice(slice):
+    ...
+
+def eat_pizza(pizza):
+    with sentry_sdk.start_transaction(op="task", name="Eat Pizza"):
+        while pizza.slices > 0:
+            eat_slice(pizza.slices.pop())
+`}
+      </StyledCodeSnippet>
+      <p>
+        {tct(
+          `For in-depth instructions on setting up tracing, view [docLink:our documentation].`,
+          {
+            docLink: <a href="https://docs.sentry.io/platforms/python/tracing/" />,
+          }
+        )}
+      </p>
+    </DocumentationWrapper>
+  );
+}
+
+function NodeDocs() {
+  return (
+    <DocumentationWrapper>
+      <p>
+        {t(
+          'To manually instrument a specific region of your code, you can create a transaction to capture it.'
+        )}
+      </p>
+      <p>
+        {tct(
+          'The following example creates a transaction for a part of the code that contains an expensive operation (for example, [code:processItem]), and sends the result to Sentry:',
+          {code: <code />}
+        )}
+      </p>
+      <StyledCodeSnippet language="javascript" dark>
+        {`app.use(function processItems(req, res, next) {
+  const item = getFromQueue();
+  const transaction = Sentry.startTransaction({
+    op: "task",
+    name: item.getTransaction(),
+  });
+
+  // processItem may create more spans internally (see next examples)
+  processItem(item, transaction).then(() => {
+    transaction.finish();
+    next();
+  });
+});
+`}
+      </StyledCodeSnippet>
+      <p>
+        {tct(
+          `For in-depth instructions on setting up tracing, view [docLink:our documentation].`,
+          {
+            docLink: (
+              <a href="https://docs.sentry.io/platforms/node/tracing/instrumentation/custom-instrumentation/" />
+            ),
+          }
+        )}
+      </p>
+    </DocumentationWrapper>
+  );
+}
+
+function ReactNativeDocs() {
+  return (
+    <DocumentationWrapper>
+      <p>
+        {t(
+          'To manually instrument a specific region of your code, you can create a transaction to capture it.'
+        )}
+      </p>
+      <StyledCodeSnippet language="javascript" dark>
+        {`const transaction = Sentry.startTransaction({ name: "test-transaction" });
+const span = transaction.startChild({ op: "functionX" }); // This function returns a Span
+// functionCallX
+span.finish(); // Remember that only finished spans will be sent with the transaction
+transaction.finish(); // Finishing the transaction will send it to Sentry
+`}
+      </StyledCodeSnippet>
+      <p>
+        {tct(
+          `For in-depth instructions on setting up tracing, view [docLink:our documentation].`,
+          {
+            docLink: (
+              <a href="https://docs.sentry.io/platforms/react-native/tracing/instrumentation/custom-instrumentation/" />
```

Comment:
- are we no longer using the `ExternalLink` component? 
- should we add here `target="_blank"`?

Reply:
Right, missed that... thx!